src/ray/object_manager/connection_pool.h

```diff
@@ -0,0 +1,148 @@
+#ifndef RAY_CONNECTION_POOL_H
```

Comment:
This isn't how we're naming include guards, right? Shouldn't it be `RAY_OBJECT_MANAGER_CONNECTION_POOL`? This applies to other files as well